librz/bin/bfile.c

```diff
@@ -860,10 +860,9 @@ static inline bool add_file_hash(RzMsgDigest *md, const char *name, RzList *list
 	return true;
 }
 
-RZ_API RzList *rz_bin_file_compute_hashes(RzBin *bin, ut64 limit) {
-	rz_return_val_if_fail(bin && bin->cur && bin->cur->o, NULL);
+RZ_API RzList *rz_bin_file_compute_hashes(RzBin *bin, RzBinFile *bf, ut64 limit) {
```

Comment:
Please add Doxygen here. Also the ownership of the return value.

Reply:
I'd leave that for a separate refactoring because these functions were already there and the PR is already quite huge with all the commands conversions to newshell. I would avoid adding other changes not strictly related to that.

Reply:
Well, the change is minimal, and it's comments. I don't think it should be in a separate PR. You changed the function signature anyway.